packages/react/src/helpers/__tests__/validateShippingAddressZipCode.test.ts

```diff
@@ -0,0 +1,134 @@
+import { type AddressType } from '@farfetch/blackout-client';
+import { validateShippingAddressZipCode } from '../index.js';
+
+const eircodeRegex =
+  '^(?=.{7,8}$)^([AC-FHKNPRTV-Y]{1}[0-9]{2}|D6W)[ ]?[0-9AC-FHKNPRTV-Y]{4}$';
+
+describe('validateShippingAddressZipCode', () => {
+  const mockAddressSchema = [
+    {
+      addressSchemaLines: [
+        {
+          id: '9a24c613-18c5-4b0d-a789-d79369777262',
+          parentId: '00000000-0000-0000-0000-000000000000',
+          name: 'Postal Code',
+          position: 0,
+          type: 'FreeText',
+          validationRegex: eircodeRegex,
+          apiMapping: 'ZipCode',
+          isMandatory: true,
+          maxLength: 45,
+          minLength: 1,
+          column: 0,
+          row: 0,
+        },
+      ],
+      addressType: 'Shipping' as AddressType,
+    },
+  ];
+
+  it("should return true if the country isn't one of the countries to validate", async () => {
+    const mockAddress = {
+      addressLine1: 'Portugal Address',
+      city: {
+        countryId: 0,
+        id: 0,
+        name: 'Portugal',
+      },
+      country: {
+        alpha2Code: 'PT',
+        id: 98,
+        name: 'Portugal',
+        continentId: 1,
+      },
+      zipCode: '4444',
+    };
+
+    expect(
+      await validateShippingAddressZipCode(mockAddress, mockAddressSchema, [
+        'ES',
+      ]),
+    ).toEqual({
+      isValid: false,
+      error: 'ER01',
```

Comment:
```suggestion
      error: ZipCodeValidationError.CountryIgnored,
```

Reply:
this error was removed, because the logic has changed and no longer makes sense.